program/src/state/config.rs

```diff
@@ -27,7 +27,7 @@ pub struct Config {
 
     /// After a deactivation, defines the number of seconds that must pass before
     /// the stake is inactive and able to be withdrawn.
-    pub cooldown_time_seconds: u64,
+    pub cooldown_time_seconds: UnixTimestamp,
```

Comment:
Are we going with `u64` everywhere? Or do you still want `i64` in state?

Reply:
I left it as `i64` on state since it is quite handy to do the arithmetic without having to convert `u64` <-> `i64` when dealing with the Clock timestamp. So the "user" input is always an `u64` – they don't need to specify a negative value.

Reply:
Ok sounds good I'm indifferent on this. If you'd rather `i64` in state, good with me!